### PR TITLE
Document stream source range operator #24933

### PR DESCRIPTION
**Pull request by davimi**
Documented range operator of Source for java DSL. #24933

### CODE REVIEW

**Review by ktoso — APPROVED** [review]
Thanks :)

**Comment by ktoso** [comment]
Compile error:

```
[error] /localhome/jenkinsakka/workspace/pr-validator-per-commit-jenkins/akka-docs/src/test/java/jdocs/stream/operators/Source.java:46:2: `}' expected but eof found.
[error] }
[error]  ^
[error] one error found
```

Please address?

File: akka-docs/src/main/paradox/stream/operators/Source/range.md [hunk 1]
```diff
@@ -4,9 +4,18 @@ Emit each integer in a range, with an option to take bigger steps than 1.
...
+  group="com.typesafe.akka"
+  artifact="akka-stream_$scala.binary_version$"
+  version="$akka.version$"
+}
+
+
 ## Description
 
-Emit each integer in a range, with an option to take bigger steps than 1.
+Emit each integer in a range, with an option to take bigger steps than 1. In Scala, you can use the apply method to generate a sequence of integers.
```
[the remark this answers is not in the document]
Comment (davimi) [reply akka-docs/src/main/paradox/stream/operators/Source/range.md]: Done

**Review by johanandren — APPROVED** [review]
LGTM, thanks!

**Pull request merged**